backend/data_fixtures/audit/excel_schema_test_files/federalawards-pass-01.json

```diff
@@ -4,37 +4,38 @@
         "total_amount_expended": 12345,
         "federal_awards": [
             {
-                "program": {
-                    "federal_agency_prefix": "42",
-                    "three_digit_extension": "RD",
-                    "additional_award_identification": 1234,
-                    "program_name": "MAIN PROGRAM",
-                    "is_major": "N",
-                    "audit_report_type": "",
-                    "number_of_audit_findings": 0,
-                    "amount_expended": 500,
-                    "federal_program_total": 4000
-                },
-                "loan_or_loan_guarantee": {
-                    "is_guaranteed": "N",
-                    "loan_balance_at_audit_period_end": 0
-                },
-                "direct_or_indirect_award": {
-                    "is_direct": "N",
-                    "entities": [
-                        {
-                            "passthrough_name": "Bob's Granting House",
-                            "passthrough_identifying_number": "12345"
-                        }
-                    ]
-                },
-                "cluster": {
-                    "cluster_name": "N/A",
-                    "cluster_total": 123
-                },
-                "subrecipients": {
-                    "is_passed": "N"
-                }
+              "program": {
+                "federal_agency_prefix": "42",
+                "three_digit_extension": "RD",
+                "additional_award_identification": 1234,
+                "program_name": "MAIN PROGRAM",
+                "is_major": "N",
+                "audit_report_type": "",
+                "number_of_audit_findings": 0,
+                "amount_expended": 500,
+                "federal_program_total": 4000
+              },
+              "loan_or_loan_guarantee": {
+                "is_guaranteed": "N",
+                "loan_balance_at_audit_period_end": 0
+              },
+              "direct_or_indirect_award": {
+                "is_direct": "N",
+                "entities": [
+                  {
+                    "passthrough_name": "Bob's Granting House",
+                    "passthrough_identifying_number": "12345"
+                  }
+                ]
+              },
+              "cluster": {
+                "cluster_name": "N/A",
+                "cluster_total": 0
+              },
+              "subrecipients": {
+                "is_passed": "N"
+              },
+              "award_index": "AWARD-00001"

```

Comment:
@jadudm Any reason award key should be a string (e.g. AWARD-00001) versus a number (e.g. 1)? I have no preference.

Reply:
We maintain the text format but reduce to 4 digit. i.e. AWARD-#### instead of AWARD-#####.